prog/bootstrap_rhizome.rb

```diff
@@ -37,11 +37,12 @@ def rootish_ssh(cmd)
     rootish_ssh(<<SH)
 set -ueo pipefail
 sudo apt update && sudo apt-get -y install ruby-bundler
+sudo userdel -rf rhizome || true
```

Comment:
Is this the only change necessary to make this idempotent? Is that the goal? Is `-rf` redundant with `|| true` (as it would be with `rm`)?

If it's one of handful of changes to do that, I'd submit it separately.